docsite/source/settings.html.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Settings
+layout: gem-single
+name: dry-system
+---
+
+## Basic usage
+
+Dry-system has already built-in settings system which you can use for your application. For this you need to use system component:
```

Comment:
```suggestion
`dry-system` provides a built-in `:settings` component which you can use in your application. In order to set it up, simply define a bootable `:settings` component and import it from the `:system` provider:
```

Reply:
@solnic do you mind if we don’t use `code-style` for our gem names in docs? They’re not technically code, they’re (unconventionally lower-cased) proper nouns.

Reply:
@timriley sure, but I cringe when I see `Dry-foo` and OTOH starting a sentence with "dry-system" looks wrong, so I dunno... 😅

Reply:
I think it’s ok to start a sentence with the lower-case, normally styled text :)

Reply:
@timriley ok great then, I'm totally ok with dropping "`" then :)